doc/source/release-notes/Percona-Server-8.0.15-6.rst

```diff
@@ -0,0 +1,64 @@
+.. rn:: 8.0.15-6
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+|Percona| announces the release of |Percona Server| |release| on |date|
+(downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from the `Percona
+Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+
+This release includes fixes to bugs found in previous releases of |Percona
+Server| 8.0.
+
+.. admonition:: Incompatible changes
+
+   In previous releases, the audit log used to produce time stamps
+   inconsistent with the ISO 8601 standard. Release |release| of
+   |Percona Server| solves this problem. This change, however, may
+   break programs that rely on the old time stamp format.
+
```

Comment:
Is this supposed to be a part of this release?  We were supposed to be doing the MyRocks backup enabling only.  I am fine if this is intentional and has come at the direction of Tyler, Peter S. Laurynas or Sveta, but if it is accidental, then we should probably remove it from the release.  I have said my peace, do with it what you think best.